tmt/tests/bootc-install-provision.sh

```diff
@@ -91,7 +95,51 @@ COPY bin /usr/local/bin
 
 # In Testing Farm, all ssh things should be reserved for ssh command run after reboot
 COPY .ssh /var/roothome/.ssh
+COMMONEOF
+
+    if [[ -v KOJI_TASK_ID ]] || [[ -v CI_KOJI_TASK_ID ]]; then
+        FEDORA_CI_CONTAINERFILE="${BOOTC_TEMPDIR}/fedora_ci_containerfile"
+        tee "$FEDORA_CI_CONTAINERFILE" > /dev/null << FEDORACIEOF
+FROM $TIER1_IMAGE_URL
+
+COPY $BOOTC_RPM_FILE_NAME /$BOOTC_RPM_FILE_NAME
+COPY $SYSTEM_REINSTALL_BOOTC_RPM_FILE_NAME /$SYSTEM_REINSTALL_BOOTC_RPM_FILE_NAME
+RUN dnf remove -y bootc system-reinstall-bootc && dnf install -y /$BOOTC_RPM_FILE_NAME /$SYSTEM_REINSTALL_BOOTC_RPM_FILE_NAME rpm-ostree
```

Comment:
Why is `rpm-ostree` mentioned here?

It'd also seem cleaner again to me to operate on `*.rpm` so adding a new subpackage wouldn't require changing a bunch of code. If we do the bind mount above I think this is could be
`RUN rpm -Uvh --oldpackage /rpms/*.rpm` or so